modules/extreme_year.py

```diff
@@ -0,0 +1,117 @@
+"""
+this module will return the highest and lowest temperatures of a
+given year along with the most humid day.
+"""
+import glob
+
+
+def read_and_calculate_extremes(pattern, path):
```

Comment:
Can you divide it into more generic functions `read_year_data(year)` that would return all the rows for an year? Probably you could filter the data based on what you only need from the file row?

Reply:
done